.github/workflows/interop.yml

```diff
@@ -147,3 +155,11 @@ jobs:
         with:
           name: ${{ matrix.alice }}_${{ matrix.bob }}_${{ matrix.recipient }}_test4.age
           path: test4.age
+
+      - name: Update FiloSottile/age status with result
+        if: github.event.action == 'age-interop-request'
+        run: |
+          curl -X POST https://api.github.com/repos/FiloSottile/age/statuses/${{ github.event.client_payload.sha }} \
+          -H 'Accept: application/vnd.github.everest-preview+json' \
+          -u ${{ secrets.AGE_STATUS_ACCESS_TOKEN }} \
+          --data '{"state": "${{ job.status }}", "target_url": "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}", "description": "Test passed!", "context": "interoperability/${{ matrix.alice }}-${{ matrix.bob }}-${{ matrix.recipient }}"}'
```

Comment:
I'll be testing the workflow once this is merged, and will make a follow-up PR with additional changes (like fixing the description once I know how it is rendered by GitHub).